docs/getting_started.md

```diff
@@ -1,14 +1,12 @@
 # Getting started
 
-At its heart Thanos provides a global query view, data backup, and access historical data as core features. All three are well separated and Thanos deployments can be adapted to make use of each of them individually. This supports using just a subset of its features as well as a gradual rollout that immediately provides some of its benefits.
+Thanos provides a global query view, data backup, and historical data access as its core features in a single binary. All three features can be run independently of each other. This allows you a subset of Thanos features for immediate benefit or testing, while also making it flexible for gradual roll outs in more complex environments. 
```

Comment:
`This allows you a subset`? 

Maybe `This allows you to have a subset..`?

Reply:
👍